app/src/main/java/com/doubtless/doubtless/main/bottomNav/RetroBottomNav.kt

```diff
@@ -0,0 +1,127 @@
+package com.doubtless.doubtless.main.bottomNav
+
+import android.content.Context
+import android.os.Bundle
+import android.os.Parcelable
+import android.os.Vibrator
+import android.os.VibratorManager
+import android.util.AttributeSet
+import android.view.View
+import android.widget.LinearLayout
+import androidx.core.view.children
+import com.doubtless.doubtless.theming.buttons.SecondaryButton
+
```

Comment:
Any specific reason why we created our own bottom nav impl rather than using existing piece?

Reply:
First of all i wanted a retro themed ui and even making some easy customisation is hard in bottom nav view by android, and i wanted to learn to do this hence I made it manually. Initially I used bottom nav only but in the way itself figured out that it'll hard to customise it.